packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/ZoomInButton/ZoomInButtonContainer.js

```diff
@@ -21,29 +21,33 @@ function storeMapper(classifierStore) {
   }
 }
 
-function ZoomInButtonContainer({ separateFrameZoomIn = () => true }) {
-  const { disabled, separateFramesView, zoomIn } = useStores(storeMapper)
```

Comment:
Removing `separateFramesView` here means it should also be removed from the `storeMapper` function in ZoomInButtonContainer and ZoomOutButton Container.